### DISCUSSION TITLE
How to enter the Matomo URL in the analytics extension?

### DISCUSSION THREAD

**Discussion by l4ndfr05ch** [datenstrom/community#965] [Ask a question] [answered]
Hi :-)

I would like to use the [yellow-analytics](https://github.com/GiovanniSalmeri/yellow-analytics) extension with Matomo. Unfortunately, a test by Matomo fails. I receive the following error message: `The verification of the tracking code has failed. Please check that you have installed the code correctly`. This error message probably appears because, according to Matomo's installation instructions, you do not have to enter the JavaScript tracking code for the Yellow system. Is that correct?

I proceeded as follows:
1. installation of Matomo via my webhoster in the subdomain matomo.landfrosch.com.
2. the installation process of Matomo was successful.
3. downloaded the zip file from the yellow-analytics extension, unzipped it and copied the three files into the extensions folder.
4. according to the instructions, I should enter the Matomo URL and siteId in the file system/extensions/yellow-system.ini. I'm not sure exactly how to enter the siteId in particular. My current entry is as follows:
`AnalyticsMatomo: https://matomo.landfrosch.com#1`

How do I enter the exact url?

Best regards,
Marko

![matomo_analytics_yellow](https://github.com/datenstrom/yellow/assets/162504154/f7fea45e-1009-469f-902e-29e7b2bdd1e0)

**Comment by schulle4u** [comment]
It looks like you didn't correctly follow the installation instructions for the extension. After downloading the zip file, **do not** extract it. Just upload the zip file into your `system/extensions` folder and open your website in the browser. Yellow will extract the files at their appropriate places, and also add all available settings for this extension into the file `system/extensions/yellow-system.ini`. The only thing left for you is to change the extension settings as described in it's documentation.

**Comment by GiovanniSalmeri** [comment]
In addition to what @schulle4u said, perhaps this page is useful for finding the siteId: https://matomo.org/faq/general/faq_19212/

Does all this solve your problem with this (experimental) extension?

**Comment by l4ndfr05ch** [comment, marked as answer]
I deleted the three analytics files which I had yesterday manually copied into the extension-folder and then uploaded the zip-file to this folder. The result had not changed. The visits to my website were not recorded. However, the cookie-window on my website was displayed. That was the result from yesterday.

Today I changed the entry in the yellow-system-ini. Now it works.

`old: AnalyticsMatomo: https://matomo.landfrosch.com#1`
`new: AnalyticsMatomo: matomo.landfrosch.com/#1`

In contrast to the first entry from yesterday, I have removed the _https://_ and added the ID after the url of my website with a slash (_/#1_). Now the visits to my website are recognised and displayed in the Matomo dashboard. 

Thank you for your support.
